Packs/Lacework/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,29 @@
+
+#### Classifiers
+##### New: Lacework - Classifier
+- Lacework Incident Classifier (Available from Cortex XSOAR 6.0.0).
+##### New: Lacework
+- 1.1.0 (Available from Cortex XSOAR 5.0.0).
```

Comment:
This line is for the 5_9_9 classifier

Reply:
Thanks again @barchen1!  I cleaned this up here: https://github.com/demisto/content/pull/15014/commits/0e730ad4e6fe6f17b48bff640fe144d830eb1fc2

The redundant versions numbers came from the `demisto-sdk update-release-notes` command - so wasn't sure if they were required or not.